src/main/java/ch/njol/skript/tests/runner/ExprParseLogs.java

```diff
@@ -0,0 +1,63 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.tests.runner;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+
+import javax.annotation.Nullable;
+
+
+public class ExprParseLogs extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprParseLogs.class, String.class, ExpressionType.SIMPLE, "[the] [last] [parse] logs");
```

Comment:
I think that  might be misleading allowing `logs` in the pattern, maybe make `parse` part mandatory would be better.
I even think `[the] [last] [parse] error` should be changed to adapt this suggestion, adding support for `error[s]`

What do you think?

Reply:
Good idea.

Reply:
> I think that might be misleading allowing `logs` in the pattern, maybe make `parse` part mandatory would be better. I even think `[the] [last] [parse] error` should be changed to adapt this suggestion, adding support for `error[s]` would be nice too.
> 
> What do you think?

so, originally it was `parse errors` however I decided it was misleading because this expression returns ALL the logs... which could include other log levels like warnings.

Reply:
Making parse mandatory makes sense to me, but I made it optional to match the existing parse error expression. Totally open to either way on that.

Reply:
IMO I would change both (this expr and parse errors) to make `parse` part mandatory however, the parse error should probably be in another PR as it's a breaking change unless you want your PR to take couple years to get merged 😎

Reply:
> IMO I would change both (this expr and parse errors) to make `parse` part mandatory however, the parse error should probably be in another PR as it's a breaking change unless you want your PR to take couple years to get merged 😎

I was aiming for a one year timeline

Reply:
What an easy task!
![image](https://user-images.githubusercontent.com/20037329/219936053-d0e87879-8b51-47c0-9a70-af96fc26dc1e.png)